ogr/services/github.py

```diff
@@ -366,21 +366,11 @@ def who_can_close_issue(self) -> Set[str]:
 
         return set(usernames)
 
-    def who_can_merge_pr(self) -> Set[str]:
-        try:
-            collaborators = self._get_collaborators_with_permission()
-        except github.GithubException:
-            logger.debug(
-                f"Current Github token must have push access to view repository permissions."
-            )
-            return set()
-
-        usernames = []
-        for login, permission in collaborators.items():
-            if permission in ["admin", "write"]:
-                usernames.append(login)
+    def who_can_close_issue(self) -> Set[str]:
+        return self.__collaborators()
```

Comment:
The name can be a little bit confusing -- I think that no all collaborators have that permission. (Also you can use some verb in the name like `get`.)

Reply:
it's given by interface... another refactor?

Reply:
I mean the `__collaborators`.

Reply:
oh ok :D